src/main/java/org/kainos/ea/resources/EmployeeController.java

```diff
@@ -29,4 +31,20 @@ public Response getAllOrders() {
             return Response.serverError().build();
         }
     }
+
+    @GET
+    @Path("/employees/delivery/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getDeliveryEmployeeById(@PathParam("id") int id) {
+        try {
+            return Response
+                    .status(Response.Status.OK)
+                    .entity(employeeService.getDeliveryEmployeeById(id))
+                    .build();
+        } catch (GenericDoesNotExistException | GenericActionFailedException e) {
```

Comment:
Do you want these exceptions to do the same thing, should they both return a `server error`?